src/app/dashboard-widgets/create-work-item-widget/create-work-item-overlay/create-work-item-overlay.component.ts

```diff
@@ -1,54 +0,0 @@
-import { AfterViewInit, Component, OnInit, ViewChild, ViewEncapsulation } from '@angular/core';
-import { Router } from '@angular/router';
-
-import { Observable } from 'rxjs';
-
-import { Broadcaster } from 'ngx-base';
-import { Space, Spaces } from 'ngx-fabric8-wit';
-
-import { removeAction } from '../../../app-routing.module';
-import { DummyService } from './../shared/dummy.service';
-
-// TODO HACK
-import { WorkItemDetailAddTypeSelectorWidgetComponent } from 'fabric8-planner/app/components/work-item-create/work-item-create-selector/work-item-create-selector.component';
-import { WorkItemType } from 'fabric8-planner/app/models/work-item-type';
-import { WorkItemService } from 'fabric8-planner/app/services/work-item.service';
-
-@Component({
-  encapsulation: ViewEncapsulation.None,
-  selector: 'fabric8-create-work-item-overlay',
-  templateUrl: './create-work-item-overlay.component.html'
-})
-export class CreateWorkItemOverlayComponent implements OnInit, AfterViewInit {
-
-  workItemTypes: Observable<any[]>;
-  space: Space;
-
-  @ViewChild('detailAddTypeSelector') overlay: WorkItemDetailAddTypeSelectorWidgetComponent;
-
-  constructor(
-    private router: Router,
-    private spaces: Spaces,
-    private broadcaster: Broadcaster,
-    private workItemService: WorkItemService
-  ) { }
-
-  ngOnInit() {
-    this.spaces.current.subscribe(space => this.space = space);
-    this.workItemTypes = this.workItemService.getWorkItemTypes();
-  }
-
-  ngAfterViewInit() {
-    this.overlay.open();
-  }
-
-  onClose() {
-    this.router.navigateByUrl(removeAction(this.router.url));
-  }
-
-  onSelect(workItemType: WorkItemType) {
-    this.router
-      .navigateByUrl(`/${this.space.relationalData.creator.attributes.username}/${this.space.attributes.name}/plan/detail/new?type=${workItemType.id}`);
-  }
-
-}
```

Comment:
Should we keep the code for the overlay? Mike K, suggested we can implement something like this "later when implementing the generic 'issue' WIT". 

Don't feel strongly about it, but thinking this component could still be useful?

Reply:
My personal preference for repository management is to only keep code that is actually executed. Commented code or code that's not executed should not exist. Granted code that implements an API for third-party users is a little different, this isn't one of those cases. Removals will exist forever in version control history for future reference. However I really don't mind either way here. Did you mean to retain this entire file? Or some portion?

Reply:
If it does become useful in the future it can always be retrieved by reverting this commit, I guess.

Reply:
Right. If you know where to look.

Also wondering if this functionality needs to be replaced with a link to Planner? That is, instead of creating a new work item, we simply navigate the user to Planner where they can create one themselves. 

I've pinged the UXD team for their input...

Reply:
The file existing doesn't make it any easier for a blind developer to know where to look though. The fastest route in both cases is to ask someone with experience for any suggestions who can point to the right place either way. If that doesn't exist, coding it from scratch isn't a tough task.